src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,95 @@
+package duke.parser;
+
+import duke.command.*;
```

Comment:
Perhaps listing each of the imports separately might make the code easier to comprehend and maintain.